manifest.json

```diff
@@ -0,0 +1,132 @@
+{
+    "app-id": "org.kiwix.Client",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "kiwix-desktop",
+    "rename-desktop-file": "kiwix-desktop.desktop",
+    "rename-icon": "kiwix-desktop",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=x11",
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-data"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/lib/cmake",
+        "/lib/*.la",
+        "/bin/curl",
+        "/bin/copydatabase",
+        "/bin/kiwix-compile-resources",
+        "/bin/quest",
+        "/bin/simple*",
+        "/bin/xapian-*",
+        "/share/aclocal",
+        "/share/doc",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "pugixml",
+            "no-autogen": false,
+            "sources": [
+                {
+                    "type": "archive",
+                    "sha256": "0f422dad86da0a2e56a37fb2a88376aae6e931f22cc8b956978460c9db06136b",
+                    "url": "http://download.kiwix.org/dev/pugixml-1.2.tar.gz",
+                    "dest": "src"
+                },
+                {
+                    "type": "patch",
+                    "path": "patches/pugixml_meson.patch"
+                }
+            ],
+            "buildsystem": "meson",
+            "builddir": true
```

Comment:
`builddir` is already default for meson, also `sources` is usually the last element; `no-autogen` defaults to `false`

Reply:
Fixed in last forced push.